code/segmentedtickerplant/pubsub.q

```diff
@@ -102,7 +100,10 @@ closesub:{[h]
 // Subscriber will call with null y parameter in sub all mode
 // In sub filtered mode, y will contain tables to subscribe to and filters to apply
 .u.sub:{[x;y]
-  if[not x in .stpps.t;'x];
+  if[not x in .stpps.t;
+    .lg.e[`rdb;"Table ",string[x]," not in list of stp pub/sub tables"];
+    :()
```

Comment:
I think you should return this error to the calling process rather than just logging an error
Give this is .u.sub, you need to mimic the return values of the old .u.sub